templates/_helpers.yaml

```diff
@@ -79,11 +79,11 @@
 {{- end }}
 
 {{ define "airflow_url" -}}
-{{ .Release.Name }}-airflow.{{ .Values.ingress.baseDomain }}
+  {{ printf "app.%s" .Values.ingress.baseDomain }}
```

Comment:
I don't think airflow should live under the same subdomain as "app"/oribit -- that gives it access to the Orbit auth cookie etc and is something we should avoid.

What do you think about a subdomain dedicated to airflow/flowers/deployments?

Reply:
@schnie do you have requirements for sub-domain name?
```
https://airflow.{{ .Values.ingress.baseDomain }}/deployments/release-name-123/airflow ?
```

Reply:
```
https://deployments.{{ .Values.ingress.baseDomain }}/d/release-name-123/airflow
https://deployments.{{ .Values.ingress.baseDomain }}/d/release-name-123/flower
```
confirmed.